plugins/jobtype/src/azkaban/jobtype/HadoopPigJob.java

```diff
@@ -86,10 +85,14 @@ public HadoopPigJob(String jobid, Props sysProps, Props jobProps, Logger log)
     obtainTokens = getSysProps().getBoolean("obtain.binary.token", false);
     userPigJar = getJobProps().getBoolean("use.user.pig.jar", false);
 
+    // Allow user to enable fetching of HCatalog tokens
+    boolean obtainHcatToken = getJobProps().getBoolean(HadoopSecurityManager.OBTAIN_HCAT_TOKEN, false);
+    getSysProps().put(HadoopSecurityManager.OBTAIN_HCAT_TOKEN, Boolean.toString(obtainHcatToken));
```

Comment:
why are we overwriting properties form private.properties. We may want to have the option of specifying this config in private.config but still over writable by user ?
 ```boolean obtainHcatToken =  getSysProps().getBoolean(HadoopSecurityManager.OBTAIN_HCAT_TOKEN, false);
obtainHcatToken = getJobProps().getBoolean(HadoopSecurityManager.OBTAIN_HCAT_TOKEN, obtainHcatToken);
 getSysProps().put(HadoopSecurityManager.OBTAIN_HCAT_TOKEN, Boolean.toString(obtainHcatToken));
```